bootcd/rpms/genesis_scripts/genesis_scripts.spec

```diff
@@ -25,45 +26,55 @@ Scripts and configuration files used by Genesis in the bootcd image
 # noop
 
 %install
-# add root's bash_profile
 mkdir -p $RPM_BUILD_ROOT/root
+# add root's bash_profile
 install -m 644 -T %{SOURCE0}   $RPM_BUILD_ROOT/root/.bash_profile.genesis_scripts
 
 # add some overrides we need
 mkdir -p $RPM_BUILD_ROOT/etc/sysconfig/network-scripts
 mkdir -p $RPM_BUILD_ROOT/etc/init
 mkdir -p $RPM_BUILD_ROOT/etc/init.d
 install -m 755 -T %{SOURCE1}   $RPM_BUILD_ROOT/etc/init.d/network-prep
-install -m 644 -T %{SOURCE2}   $RPM_BUILD_ROOT/etc/sysconfig/init.diff
-install -m 644 -T %{SOURCE3}   $RPM_BUILD_ROOT/etc/init/tty.conf.override
+install -m 755 -T %{SOURCE6}   $RPM_BUILD_ROOT/etc/init.d/genesis
+install -m 644 -T %{SOURCE8}   $RPM_BUILD_ROOT/etc/sysconfig/genesis
 
 # add helper for agetty
-install -m 555 -T %{SOURCE5}   $RPM_BUILD_ROOT/root/login-shell
-
-# add the bootloader
 mkdir -p $RPM_BUILD_ROOT/usr/bin/
+install -m 555 -T %{SOURCE5}   $RPM_BUILD_ROOT/usr/bin/autologin
+
+# add the bootloader and its wrapper
 install -m 555 -T %{SOURCE4}   $RPM_BUILD_ROOT/usr/bin/genesis-bootloader
+install -m 755 -T %{SOURCE7}   $RPM_BUILD_ROOT/usr/bin/run-genesis-bootloader
 
 %clean
 # noop
 
 %files
 %defattr(-, root, root)
+%config /etc/init.d/genesis
```

Comment:
This isn't a config, just a file. Replacement​ behavior is different with configs in rpm